src/cpp/fastdds/domain/DomainParticipantImpl.hpp

```diff
@@ -332,7 +332,7 @@ class DomainParticipantImpl
 
     DomainId_t get_domain_id() const;
 
-    ReturnCode_t delete_contained_entities();
+    virtual ReturnCode_t delete_contained_entities();
```

Comment:
Does this break ABI compatibility?

Reply:
Mmm not really since it only affects inner Impl (I think), is that correct @MiguelCompany ?

Reply:
Correct. This only affects internal implementation, so no ABI break.